packages/usehooks-ts/src/useDocumentTitle/useDocumentTitle.ts

```diff
@@ -1,7 +1,17 @@
+import { useRef } from 'react'
+
 import { useIsomorphicLayoutEffect } from '..'
 
-export function useDocumentTitle(title: string): void {
+export function useDocumentTitle(title: string, prevailOnUnmount = true): void {
+  const defaultTitle = useRef(window.document.title)
+
   useIsomorphicLayoutEffect(() => {
     window.document.title = title
   }, [title])
+
+  useIsomorphicLayoutEffect(() => () => {
```

Comment:
No need to use `useLayoutEffect` to unmount logic. 
We recently released an `useUnmount` hook perfect for this use-case